shell-ui/src/Navbar.spec.js

```diff
@@ -123,12 +123,59 @@ describe('navbar', () => {
             redirect-url="http://localhost:8082"
             config-url="/shell/config.json"
             scopes="openid profile email groups offline_access audience:server:client_id:oidc-auth-client"
+            options={
+                JSON.stringify({
+                    "main": {
+                        "http://localhost:8082/":{ "en": "Platform", "fr": "Plateforme" },
+                        "http://localhost:8082/test":{ "en": "Test", "fr": "Test" }
+                    },
+                    "subLogin": {}
+                })
+            }
+            />
+        )
+        //E
+        await waitForLoadingToFinish();
+        //V
```

Comment:
Are those comments meaning something? Never seen them before

Reply:
I used them to segregate my tests in 3 steps : 
 - //S = //Setup, set all the tests dependencies
 - //E = //Exercise, act / perform the action which is being tested
 - //V = //Verify, assert on the result of the action